src/mod.ts

```diff
@@ -0,0 +1,7 @@
+export * from './types.ts';
```

Comment:
With this the test files should be able to `import { ... }` everything from `../src/mod.ts`, instead of having to “shop around” the entire `src/` directory.

Reply:
Yep! I don't know if `mod.ts` is complete, I suppose not... it's the bare minimum for testing out LionCore in Tylasu, including [serialization/deserialization](https://github.com/Strumenta/tylasu/blob/63b3e3e7664c199227c34e36d79513e674f99188/tests/lionweb.test.ts#L84).

Reply:
I think we should add “everything”/as much as possible to `mod.ts` for this to make sense.